app/admin/events.rb

```diff
@@ -79,4 +86,15 @@
     end
     active_admin_comments
   end
+
+  sidebar 'Featured Projects', only: :show do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.